joint_limits_interface/test/joint_limits_urdf_test.cpp

```diff
@@ -14,12 +14,12 @@
 
 /// \author Adolfo Rodriguez Tsouroukdissian
 
-#include <gtest/gtest.h>
```

Comment:
unsolicited change :D

Reply:
It's only here to solve the warning given by pre-commit